Analyzer/include/Config.h

```diff
@@ -311,20 +312,20 @@ class Config
         else if (analyzer=="AnalyzeTest")
         {
             const std::vector<std::string> modulesList = {
-                "PrepNTupleVars",
+                //"PrepNTupleVars",
```

Comment:
Uncomment these modules for the default test analyzer. Do you have a good reason to comment these out?